pkg/mixin/runner.go

```diff
@@ -50,10 +50,10 @@ func (r *Runner) Validate() error {
 
 func (r *Runner) Run() error {
 	if r.Debug {
-		fmt.Fprintf(r.Out, "DEBUG mixin:    %s\n", r.Mixin)
-		fmt.Fprintf(r.Out, "DEBUG mixinDir: %s\n", r.mixinDir)
-		fmt.Fprintf(r.Out, "DEBUG file:     %s\n", r.File)
-		fmt.Fprintf(r.Out, "DEBUG stdin:\n%s\n", r.Step)
+		fmt.Fprintf(r.Err, "DEBUG mixin:    %s\n", r.Mixin)
```

Comment:
Similar question as posed above; wondering if informational (non-error) debug messages should in fact be sent to stderr?